soccer/gameplay/plays/testing/test_capture.py

```diff
@@ -33,8 +34,13 @@ def __init__(self):
             'robot away from ball')
 
         self.add_transition(
-            TestCapture.State.capturing, TestCapture.State.setup,
+            TestCapture.State.capturing, TestCapture.State.rotating,
```

Comment:
Could rotation be gated behind a  boolean at the top of the file? I think this might get annoying on the field.

You could do this by just going into/out of rotation in a single frame, to keep things simple.

Reply:
I'm not sure if this would really be annoying during comp since it's almost impossible to tell if capture is actually working without the aim added.